webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
                 "countsTowardScore": false
             },
             "interop-2022-dialog": {
                 "description": "Dialog Element",
                 "mdn": "https://developer.mozilla.org/docs/Web/HTML/Element/dialog",
                 "spec": "https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element",
-                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-dialog", 
+                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-dialog",
                 "countsTowardScore": false
             },
             "interop-2022-text": {
                 "description": "Typography and Encodings",
                 "mdn": "",
                 "spec": "",
-                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-text", 
+                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-text",
                 "countsTowardScore": false
             },
             "interop-2022-viewport": {
                 "description": "Viewport Units",
                 "mdn": "",
                 "spec": "https://drafts.csswg.org/css-values/#viewport-relative-units",
-                "tests": "/results/css/css-values?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-viewport", 
+                "tests": "/results/css/css-values?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-viewport",
                 "countsTowardScore": false
             },
             "interop-2022-webcompat": {
                 "description": "Web Compat",
                 "mdn": "",
                 "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-webcompat", 
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-webcompat",
                 "countsTowardScore": false
             },
             "interop-2023-cssborderimage": {
                 "description": "Border Image",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-image",
+                "spec": "https://w3c.github.io/csswg-drafts/css-backgrounds/#the-border-image",
```

Comment:
nit: Some links are `drafts.csswg.org/` instead of `w3c.github.io/csswg-drafts/`. Even though they should mirror each other, it would be nice to stay consistent.

(Ditto for the rest of the PR)

Reply:
updated